cmd/minishift/cmd/completion.go

```diff
@@ -73,6 +73,9 @@ var completionCmd = &cobra.Command{
 	Use:   "completion SHELL",
 	Short: "Outputs minishift shell completion for the given shell (bash or zsh)",
 	Long:  longDescription,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		//noop
```

Comment:
@anjannath  can you remove the //noop comments as I do not think we need a comment here or I can change these and merge. Let me know

Reply:
@LalatenduMohanty @anjannath I would like to keep it to remind myself that is no operation being performed in this block.
We have such scenarios in other project as well. 
https://github.com/search?q=%2F%2F+noop&type=Code

Update it as `// noop` :wink:

Reply:
If you strongly feel like keeping it then please follow the proper indent while making a comment.